Add Beatmap Spotlights: Winter 2020 newspost (#3337)
ppy/osu-wiki: news/2020-03-20-beatmap-spotlights-winter-2020.md

---
layout: post
title: "Beatmap Spotlights: Winter 2020"

> date: 2020-03-20 08:00:00 +0000

Reviewer: ```suggestion
date: 2020-03-31 08:00:00 +0000
```

Reviewer: Please re-date everything to this date. Our target date is always either the last day of the month or the first day of the next.

*written by [Ulqui](https://osu.ppy.sh/users/1263669)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/taiko/1000.jpg)](https://osu.ppy.sh/beatmapsets/1032172)

Coming in third this season, we have the creatively named 1000 gecs by the artist 100 gecs. Please beware, this is an extremely long map by [Genjuro](https://osu.ppy.sh/users/3196091), sporting 23 minutes of silly songs backed by a solid and diverse map! If you have half an hour to spare, it will not be wasted on playing this map, which will surely put a smile on your face not only due to the song choice, but also due to how they are mapped. This map is clearly a work of passion from Genjuro!

*written by [Nepuri](https://osu.ppy.sh/users/6637817)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/taiko/hibi.jpg)](https://osu.ppy.sh/beatmapsets/1006568)

> Crazy snappings? Stupidly fast sections? A ton of finishers? SV to strain your eyes? This map has it all. In 4th place we have KikuoHana - Hibi Darake no Me by [Alchyr](https://osu.ppy.sh/users/4993032). Not many people can equal a crazy song with a crazy map, but this guy is one of the best at it. With three difficulties to cater for more experienced to the most experienced players, it's not hard to tell the mapper has put in a lot of work into this mapset and to make it a nightmare to FC. The map starts off with the first half being deceivingly calm, until the drop hits and the map explodes into a burst of fun. All difficulties (with the top diff in particular) utilise a wide range of tough patterns constantly varying in length, rhythm and snappings which prove to be difficult for even the best of players, and it's one and a half minutes of hell which shows how creative mappers, especially Alchyr, are getting with their work and how high the bar for map difficulty has been set.

Reviewer: ```suggestion
Crazy snappings? Stupidly fast sections? A ton of finishers? SV to strain your eyes? This map has it all. In 4th place we have KikuoHana - Hibi Darake no Me by [Alchyr](https://osu.ppy.sh/users/4993032). 

Not many people can equal a crazy song with a crazy map, but this guy is one of the best at it. With three difficulties to cater for more experienced to the most experienced players, it's not hard to tell the mapper has put in a lot of work into this mapset and to make it a nightmare to get a full combo. 

The map starts off with the first half being deceivingly calm, until the drop hits and the map explodes into a burst of fun. All difficulties (with the top diff in particular) utilise a wide range of tough patterns constantly varying in length, rhythm and snappings which prove to be difficult for even the best of players, and it's one and a half minutes of hell which shows how creative mappers, especially Alchyr, are getting with their work and how high the bar for map difficulty has been set.
```

*written by [\_DUSK\_](https://osu.ppy.sh/users/6092181)*

## <a id="catch"></a> osu!catch

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/chocolate.jpg)](https://osu.ppy.sh/beatmapsets/1061800)

> Spring is coming, wanna play a map with spring flavor? hot chocolate by [JBHyperion](https://osu.ppy.sh/users/4879508) should be one of the best choices here. Needless to say, the high quality mapping style is always a mark for featured artist maps, this map is no exception. Despite the extraordinary clean and fresh flows, the mapper also tried some new features here - extending sliders and 1/8 hyperdashes. These innovative patterns bring the map to the next level and make the map stands out as well. Please make a cup of hot chocolate and enjoy this fantastic map!

Reviewer: ```suggestion
Spring is coming, wanna play a map with spring flavour? 'hot chocolate' by [JBHyperion](https://osu.ppy.sh/users/4879508) should be one of the best choices here. Needless to say, the high-quality mapping style is always a mark for featured artist maps, this map is no exception. 

Despite the extraordinary clean and fresh flows, the mapper also tried some new features here - extending sliders and 1/8 Hyperdashes. These innovative patterns bring the map to the next level and make the map stands out as well. Please make a cup of hot chocolate and enjoy this fantastic map!
```

*written by [Yumeno Himiko](https://osu.ppy.sh/users/1806962)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/loving.jpg)](https://osu.ppy.sh/beatmapsets/1080025)

> 'How on earth did eroge get into spotlights?' may be a question that pops up when first encountering this set. Despite the general stigma against the genre, [Luminate](https://osu.ppy.sh/users/4778689)'s "Loving Trip" does a great job of following the song. Slower parts of the song keep things interesting with varied rhythm and patterns, while higher density sections have liberal use of antiflow that contrasts well. This is one of the few sets that I've genuinely had fun playing this year, and I hope you give it a shot as well.

Reviewer: ```suggestion
'How on earth did eroge get into spotlights?' may be a question that pops up when first encountering this set. Despite the general stigma against the genre, [Luminate](https://osu.ppy.sh/users/4778689)'s "Loving Trip" does a great job of following the song. 

Slower parts of the song keep things interesting with varied rhythm and patterns, while higher density sections have liberal use of antiflow that contrasts well. This is one of the few sets that I've genuinely had fun playing this year, and I hope you give it a shot as well.
```

*written by [Sinnoh](https://osu.ppy.sh/users/4236057)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/lights.jpg)](https://osu.ppy.sh/beatmapsets/1044462)

> [Kukkai](https://osu.ppy.sh/users/7811952)'s "Lights of Muse" has a certain feeling behind it that is fairly uncommon these days, reminiscent of something that would have easily had a play count in the hundred thousands if it was ranked a few years ago. While use of patterns and rhythm is fairly simple, the whole set is at a consistent level of fun that is hard to come by.

Reviewer: ```suggestion
[Kukkai](https://osu.ppy.sh/users/7811952)'s "Lights of Muse" has a certain feeling behind it that is fairly uncommon these days, reminiscent of something that would have easily had a play count in the hundred thousand if it was ranked a few years ago. While the use of patterns and rhythm is fairly simple, the whole set is at a consistent level of fun that is hard to come by.
```

*written by [Kurokami](https://osu.ppy.sh/users/260933)*

## <a id="mania"></a> osu!mania

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/mania/transhuman.jpg)](https://osu.ppy.sh/beatmapsets/1069426)

> Incredibly exciting yet surprisingly delicate in its structure, [Kamuy](https://osu.ppy.sh/users/7439226)'s Transhumanist stands out among currently ranked maps by being the first semi-dump ranked in osu!mania. This map has been making waves in both mapping as well as higher-level playing communities by combining consistent usage of pattern/snap arrangements to represent specific kinds of wubs in the song in order to fully represent intensity with player-friendly yet challenging patterns. These make it an absolute joy to play through and fascinating to pick apart in the editor as well. Even though it's the first of it's kind in the ranked section, Transhumanist sets an extremely high bar and proves that these kinds of maps are not only rankable, but exceptional in terms of mapping quality and providing an enjoyable player experience. This map is arguably one of Kamuy's best maps so far and we're proud to showcase it in spotlights!

Reviewer: ```suggestion
Incredibly exciting yet surprisingly delicate in its structure, [Kamuy](https://osu.ppy.sh/users/7439226)'s Transhumanist stands out among currently ranked maps by being the first semi-dump ranked in osu!mania. 

This map has been making waves in both mapping as well as higher-level playing communities by combining consistent usage of pattern/snap arrangements to represent specific kind of wubs in the song in order to fully represent intensity with player-friendly yet challenging patterns. These make it an absolute joy to play through and fascinating to pick apart in the editor as well. 

Even though it's the first of its kind in the ranked section, Transhumanist sets an extremely high bar and proves that these kinds of maps are not only rankable but exceptional in terms of mapping quality and providing an enjoyable player experience. This map is arguably one of Kamuy's best maps so far and we're proud to showcase it in spotlights!
```

*written by [-MysticEyes](https://osu.ppy.sh/users/6253266)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/mania/experiment.jpg)](https://osu.ppy.sh/beatmapsets/914341)

> [Curiossity](https://osu.ppy.sh/users/6490035) strikes again with another insanely innovative and creative masterpiece. Every bit of this map is thought out with very particularly detailed and visually appealing patterning and SV, which gives the map a great amount of replay value, especially for a marathon. The map's excitement is maintained throughout the map, making for a roller-coaster of an experience. It never gets boring, not even during the slower sections, in which the player must pay very close attention to the LN releases at a slow scroll speed. This map thus deserves its top spot in the spotlights, and we hope to see Curiossity continue to produce spectacular maps like this one in the future.

Reviewer: ```suggestion
[Curiossity](https://osu.ppy.sh/users/6490035) strikes again with another insanely innovative and creative masterpiece.

Every bit of this map is thought out with very particularly detailed and visually appealing patterning and speed changes, which gives the map a great amount of replay value, especially for a marathon. The map's excitement is maintained throughout the map, making for a roller-coaster of an experience. 

It never gets boring, not even during the slower sections, in which the player must pay very close attention to the long note releases at a slow scroll speed. This map thus deserves its top spot in the spotlights, and we hope to see Curiossity continue to produce spectacular maps like this one in the future.
```

*written by [-mint-](https://osu.ppy.sh/users/8976576)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/mania/every.jpg)](https://osu.ppy.sh/beatmapsets/905599)

> If you had to describe this map with one word, it'd be something along the lines of... Intricate. Detailed. Accurate. Clean. Recently we have seen a rise of mappers in osu!mania with this approach in mind, [Monheim](https://osu.ppy.sh/users/6814203) and [Vortex-](https://osu.ppy.sh/users/4999669) being well-known for their precision and accuracy in their maps. This map in particular accomplishes things that many mappers most likely would not be able to accomplish if they were to map this - it is musically accurate in that it expresses many instruments and melody lines at once, is delicate and clean to play, and provides a challenge for all sorts of players. These remain consistent through each difficulty of the mapset. This map thus deserves its spot in this season's spotlights, and be on the lookout - we most definitely will see more by these two extraordinarily talented mappers in the future.

Reviewer: ```suggestion
If you had to describe this map with one word, it'd be something along the lines of... Intricate. Detailed. Accurate. Clean. Recently we have seen a rise of mappers in osu!mania with this approach in mind, [Monheim](https://osu.ppy.sh/users/6814203) and [Vortex-](https://osu.ppy.sh/users/4999669) being well-known for their precision and accuracy in their maps. 

This map, in particular, accomplishes things that many mappers most likely would not be able to accomplish if they were to map this - it is musically accurate in that it expresses many instruments and melody lines at once, is delicate and clean to play, and provides a challenge for all sorts of players. These remain consistent through each difficulty of the mapset. This map thus deserves its spot in this season's spotlights, and be on the lookout - we most definitely will see more by these two extraordinarily talented mappers in the future.
```

*written by [-mint-](https://osu.ppy.sh/users/8976576)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/mania/oceanus.jpg)](https://osu.ppy.sh/beatmapsets/539179)

> It has been a while before [Level 51](https://osu.ppy.sh/users/3617847) made his appearance in mapping ground, and this time he brings the community with a great 6k mapset, Oceanus. GALE_FORCE, as the top difficulty of the mapset, proposes a real challenge for advanced players with its dense but well-balanced streams that follow the shifts in music. On the other hand, the lower two difficulties definitely provide great opportunities for general players to start with 6K and practice over 6K streams as the mapper managed to effectively balance the density within and between these difficulties. Hitsounds are designated in a way that reasonably support musical rhythms but do not distract players from focusing on the streams. That is to say, Oceanus deserves a place in spotlight as a recommendation to every player interested in progressing and challenging with 6K maps.

Reviewer: ```suggestion
It has been a while before [Level 51](https://osu.ppy.sh/users/3617847) made his appearance in mapping ground, and this time he brings the community with a great 6k mapset, Oceanus. 

GALE_FORCE, as the top difficulty of the mapset, proposes a real challenge for advanced players with its dense but well-balanced streams that follow the shifts in the music. On the other hand, the lower two difficulties definitely provide great opportunities for general players to start with 6K and practice over 6K streams as the mapper managed to effectively balance the density within and between these difficulties. 

Hitsounds are designated in a way that reasonably supports musical rhythms but does not distract players from focusing on the streams. That is to say, Oceanus deserves a place in the Spotlights as a recommendation to every player interested in progressing and challenging with 6K maps.
```

*written by [Shima Rin](https://osu.ppy.sh/users/6089608)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/mania/altros.jpg)](https://osu.ppy.sh/beatmapsets/877895)

> It has been about 3 years since [ByeForNow](https://osu.ppy.sh/users/7199159) has had a ranked map, and he comes back with an absolutely spectacular set! Despite being only three difficulties, each difficulty manages to have its own solid structure where instruments are represented through columns instead of the usual chords. You might want to turn on the hitsounds as well because the set is beautifully keysounded. Although keysounded, the set has high focus on the overall pitch and ByeForNow nails it perfectly! Delicate and precise, ByeForNow's Altros deserves a spot in this season's spotlights, and we will hope to see what other fantastic maps he produces!

Reviewer: ```suggestion
It has been about 3 years since [ByeForNow](https://osu.ppy.sh/users/7199159) has had a ranked map, and he comes back with an absolutely spectacular set! 

Despite being only three difficulties, each difficulty manages to have its own solid structure where instruments are represented through columns instead of the usual chords. You might want to turn on the hitsounds as well because the set is beautifully keysounded. Although keysounded, the set has a high focus on the overall pitch and ByeForNow nails it perfectly! 

Delicate and precise, ByeForNow's Altros deserves a spot in this season's Spotlights, and we will hope to see what other fantastic maps he produces!
```

Though the map may seem quite simple at first glance, it soon picks up the pace (literally). The seemingly simple mapping concepts complements the constantly accelerating music, representing the same melody with similar rhythm choices, but with the increased tempo comes with increased strain, effectively showcasing the intensity, while keeping the gameplay fair for players. Even at the end, when the music gets dissolved by noise, the map looks out for the drowned melody.

With such a well-crafted mapset, you should definitely try this out!

*written by [KnightC0re](https://osu.ppy.sh/users/7894340)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/osu/umineko.jpg)](https://osu.ppy.sh/beatmapsets/998836)

Bringing back the nostalgic memories, six "traditional" mappers - [Leader](https://osu.ppy.sh/users/631530), [\-kevincela\-](https://osu.ppy.sh/users/266596), [Mirash](https://osu.ppy.sh/users/2841009), [Okoratu](https://osu.ppy.sh/users/1623405), [Rue](https://osu.ppy.sh/users/417551) and [Nemis](https://osu.ppy.sh/users/1635091) - collaborated on an insanely amazing Umineko no Naku Koro ni project. 

> It includes 8 remixed songs by zts, an old friend of us, and was brought up by La Cataline, another well-known name in our community. This 30-minute long mapset basically concludes the whole flow-based mapping scene: nearly every pattern in this map maximizes the playability and readability by using intuitive flow techniques to create an unforgettable playing experience. Besides, even though 6 mappers included have different aesthetic choices, each of their styles combined with others surprisingly well while still stands out individually to give players consistent and frequent visual stimulations. 

Reviewer 2: i wondered if that word is way too exaggerated lol

```suggestion
It includes 8 remixed songs by zts, an old friend of us, and was brought up by La Cataline, another well-known name in our community. Leading by the beloved flow mapping style, nearly every pattern in this 30-minute long mapset maximizes the playability and readability by using intuitive flow techniques to create an unforgettable playing experience. Besides, even though 6 mappers included have different aesthetic choices, each of their styles combined with others surprisingly well while still stands out individually to give players consistent and frequent visual stimulations.
```

This map has only 4.84* as its star rating, which means it can be enjoyed by the majority of the players - so if you are searching for a wonderful experience that lasts as long as possible, this compilation could be one of your rare best choices. Side note: if you have played Umineko no Naku Koro ni, please enable the storyboard made by [tokiko](https://osu.ppy.sh/users/2836455), which summarizes its whole story in a nutshell.

*written by [Ryuusei Aika](https://osu.ppy.sh/users/7777875)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/osu/xaaa.jpg)](https://osu.ppy.sh/beatmapsets/999260)

> A harrowing, unsettling, some might say even cursed or creepy experience awaits whoever dares to come close to this map. [Hey lululu](https://osu.ppy.sh/users/4086497) creates the complete package of an experience from storyboard to custom skin for XaaaCi. by Mitose Noriko and with an expressive artistic style which will leave an impact well into your nightmares. The mapping takes it's roots the elements surrounding it in a ubsetteling odd manner, just to throw you off even more!

Reviewer 3: yes

> To maximize the effect of this map, enable everything from storyboard and hitsounds to the custom skin. However, player discression is advised and we take no responsibility for any lasting psychological damages!

Reviewer 3: ```suggestion
To maximize the effect of this map, make sure to enable everything in the set, from the storyboard and the hitsounds to the custom skin. Player discretion is advised and we take no responsibility for any lasting psychological damage!
```

This masterfully designed map is an experience that you don’t want to miss!

*written by [KnightC0re](https://osu.ppy.sh/users/7894340)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/osu/sealer.jpg)](https://osu.ppy.sh/beatmapsets/1067056)

[celerih](https://osu.ppy.sh/users/4696296)'s idiosyncratic take on free-formness has always stood out from the ocean of ''flow-oriented'' maps, and this rendition of Zektbach - The Sealer ~A Milia to Milia no Min~ encapsulates this philosophy while showcasing one of their boldest palette collection. 

A concise mapset content-wise, the folkish ballet is designed with engaging rhythms and cutting edge patterns with meticulous attention to feedback, all to synthesize a delightful experience of a dance and a strong, cohesive character that one will scarcely discover from a map.

*written by [faygo](https://osu.ppy.sh/users/5199686)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/osu/felys.jpg)](https://osu.ppy.sh/beatmapsets/1056144)

> Refreshing the CS5 mapping atmosphere, [Necho](https://osu.ppy.sh/users/4086593) brought the old simple yet spicy techniques introduced by tsuka, kiddly and Kloyd, make flexible use of them then create another simple yet spicy mapset on his new mapset, onoken's "felys". Differing from the classic Shiirn one, Necho used several representatives "old-school" cs5 mapping styles, including consistent 3D overlaps on red node sliders, hugely curved short sliders and instantly visible object connections at every second through the whole map. 

Reviewer 4: ```suggestion
Refreshing the CS5 mapping atmosphere, [Necho](https://osu.ppy.sh/users/4086593) brought the old simple, yet spicy techniques introduced by tsuka, kiddly and Kloyd and made flexible use of them to create another simple, yet spicy mapset on his latest work set to onoken's "felys". Differing from Shiirn's classic take, Necho used several representative "old-school" CS5 mapping styles, including consistent 3D overlaps on red node sliders, hugely curved short sliders and instantly visible object connections throughout the whole map. 
```

> All of these combines with the minimalism storyboard by Necho himself and remix-tier hitsound by Otosaka-Yu (Suzuki_1112) results in a unique - or "illegal" experience in the current mapping scene. If you are asking "where are the stunning old cs5 mappers go nowadays", or "where can I find the most epic hitsounds" this could be one of the closest answers for both.

Reviewer 4: ```suggestion
All of these combine with a minimalistic storyboard by Necho himself and remix-tier hitsounding by Otosaka-Yu (Suzuki_1112), resulting in a unique (or "illegal") experience in the current mapping scene. If you've asked "where have the stunning old CS5 mappers gone?", or "where can I find the most epic hitsounding imaginable", this set could be your answer at long last!
```

*written by [Ryuusei Aika](https://osu.ppy.sh/users/7777875)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/osu/precious.jpg)](https://osu.ppy.sh/beatmapsets/874732)

> TAKE FLIGHT! Almost quite literally with [Vell](https://osu.ppy.sh/users/3414541)'s map on Chata's Precious Wings!

Reviewer 4: ```suggestion
TAKE FLIGHT (almost quite literally) with [Vell](https://osu.ppy.sh/users/3414541)'s map set to Chata's Precious Wings!
```

Reviewer 3: ```suggestion
TAKE FLIGHT (almost quite literally) with [Vell](https://osu.ppy.sh/users/3414541)'s map seont to Chata's Precious Wings!
```
i want the "on" cause of the pun i am going here. Also not sure if brackets are a good idea for a newspost but if it's ok to have them it should be g.

Reviewer 4: brackets are fine!

Vell wears his heart on the sleeve, or rather the editor, with this one as you experience the first ever aviation training map in osu! history! 

Practice taking off and landing gracefully as the map takes flight itself and reach the skies! If the concept isn't quite clear to you yet, you may just try it out yourself.

But beware!

> No parachutes attached.

Reviewer 4: need to smoosh this one into 1-2 paragraphs instead of multiple lines

*written by [Smokeman](https://osu.ppy.sh/users/2140676)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/osu/haryo)](https://osu.ppy.sh/beatmapsets/576232)

> Being a slightly too early bird, [milr_](https://osu.ppy.sh/users/4485933) created a wonderful mapset for Azu’s Haryu, the song of the dragon of spring (not winter).

Reviewer 4: ```suggestion
Being a slightly too early bird, [milr_](https://osu.ppy.sh/users/4485933) has created a wonderful mapset for Azu’s Haryu, the song of the dragon of spring (not winter).
```

> The highest difficulty, mapped by Down, features the conventional and effective patterning. Prioritizing aesthetics, they made sure that every pattern is equidistant to one another, making the map extremely clean and well-presented. milr_ range of difficulties should also be highlighted. Although still pleasing to the eye, the map shines with its movement concepts. With a lot of sliders pointing away from their destinations, the map allows movement to be vastly varied without the constraint of implied flow. This works extremely well with the abundance of round slider shapes. This visual and movement concept is implemented across lower difficulties, allowing a bigger part of the playerbase to experience this niche idea.

Reviewer 4: ```suggestion
The highest difficulty, mapped by Down, features the conventional and effective patterning. Prioritizing aesthetics, they made sure that every pattern is equidistant to one another, making the map extremely clean and well-presented. milr_'s range of difficulties are not beyond note - they remain pleasing to the eye and truly shine in exemplifying well-thought out movement design. This couples extremely well with an abundance of round slider shapes. This visual and movement concept is implemented across lower difficulties, allowing a bigger part of the playerbase to experience this niche idea.
```

Try this map and spice up your collection!

*written by [KnightC0re](https://osu.ppy.sh/users/7894340)*

## <a id="taiko"></a> osu!taiko

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/taiko/leviathan.jpg)](https://osu.ppy.sh/beatmapsets/1013884)

Topping off this season's Spotlights, we have a map that keeps you busy with complicated rhythms all the way through - Leviathan, a mostly piano-based Touhou arrangement, mapped by [yassu-](https://osu.ppy.sh/users/7095592), a mapper who's shown their skill at handling rhythmically complicated songs time and time again! 

In this map specifically, a lot of the complexity comes from blue tick rhythms and tricky off-hand patterns mixed with 1/6 and 1/8 for even more handswitches whenever the piano calls for it, as well as some standalone 1/8 doubles sneaked in which feel really pleasant to play - all in all a great fit for spotlights, even more so as the set as a whole manages to maintain its rhythmical complexity across the lower difficulties, making it a treat for people at all levels seeking a challenge!

> *written by [Greenshell](https://osu.ppy.sh/users/8693851)*

Reviewer 4: greenshell goodshell

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/taiko/dungeon.jpg)](https://osu.ppy.sh/beatmapsets/1056653)

> On second place we have sakuraburst - dungeon music by [IceOC](https://osu.ppy.sh/users/5482401). Outstanding SV map that can be a tough challenge for any player despite its simplicity and low star rating. 

Reviewer 4: ```suggestion
In second place, we have [IceOC](https://osu.ppy.sh/users/5482401)'s take on sakuraburst - dungeon music. This is an outstanding SV map that can be a tough challenge for any player despite its simplicity and low star rating. 
```

IceOC shows how much details this map has with variations from x0.13 to x10.00 slider velocity, applied on high-speed ninja finishers, extremely slow sections and tricky patterns, everything attached to an exceptional representation of music in an entertaining and interesting way. Such uniqueness and clean execution make this a well-deserved spotlight.

*written by [Ulqui](https://osu.ppy.sh/users/1263669)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/taiko/1000.jpg)](https://osu.ppy.sh/beatmapsets/1032172)

> Coming in third this season, we have the creatively named 1000 gecs by the artist 100 gecs. 

Reviewer 4: ```suggestion
Coming in third this season, we have the creatively named 1000 gecs by the artist 100 gecs, mapped by [Genjuro](https://osu.ppy.sh/users/3196091).
```

> Please beware, this is an extremely long map by [Genjuro](https://osu.ppy.sh/users/3196091), sporting 23 minutes of silly songs backed by a solid and diverse map! If you have half an hour to spare, it will not be wasted on playing this map, which will surely put a smile on your face not only due to the song choice, but also due to how they are mapped. This map is clearly a work of passion from Genjuro!

Reviewer 4: ```suggestion
Please beware - this is an extremely long map, sporting 23 minutes of silly songs backed by a solid and diverse map! If you have half an hour to spare, it will not be wasted on playing this map, which will surely put a smile on your face not only due to the song choice, but also due to how they are mapped. This map is clearly a work of passion from Genjuro!
```

*written by [Nepuri](https://osu.ppy.sh/users/6637817)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/taiko/hibi.jpg)](https://osu.ppy.sh/beatmapsets/1006568)

Crazy snappings? Stupidly fast sections? A ton of finishers? SV to strain your eyes? This map has it all. In 4th place we have KikuoHana - Hibi Darake no Me by [Alchyr](https://osu.ppy.sh/users/4993032). 

> Not many people can equal a crazy song with a crazy map, but this guy is one of the best at it. With three difficulties to cater for more experienced to the most experienced players, it's not hard to tell the mapper has put in a lot of work into this mapset and to make it a nightmare to get a full combo. 

Reviewer 4: ```suggestion
Not many people can equal a crazy song with a crazy map, but this guy is one of the best at it. With three difficulties to cater for more experienced to the most advanced players, it's not hard to tell the mapper has put in a lot of work into this mapset to make a full combo an achievement to be proud of.
```

The map starts off with the first half being deceivingly calm, until the drop hits and the map explodes into a burst of fun. All difficulties (with the top diff in particular) utilise a wide range of tough patterns constantly varying in length, rhythm and snappings which prove to be difficult for even the best of players, and it's one and a half minutes of hell which shows how creative mappers, especially Alchyr, are getting with their work and how high the bar for map difficulty has been set.

> *written by [\_DUSK\_](https://osu.ppy.sh/users/6092181)*

Reviewer 4: up there with greenshell's one for best review comment so far, good work

## <a id="catch"></a> osu!catch

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/chocolate.jpg)](https://osu.ppy.sh/beatmapsets/1061800)

Spring is coming, wanna play a map with spring flavour? 'hot chocolate' by [JBHyperion](https://osu.ppy.sh/users/4879508) should be one of the best choices here. Needless to say, the high-quality mapping style is always a mark for featured artist maps, this map is no exception. 

> Despite the extraordinary clean and fresh flows, the mapper also tried some new features here - extending sliders and 1/8 Hyperdashes. These innovative patterns bring the map to the next level and make the map stands out as well. Please make a cup of hot chocolate and enjoy this fantastic map!

Reviewer 4: ```suggestion
Despite the extraordinarily clean and fresh flows, the mapper also tried some new features here - extending sliders and 1/8 hyperdashes. These innovative patterns bring the map to the next level and make the map stands out as well. Please make a cup of hot chocolate and enjoy this fantastic map!
```

*written by [Yumeno Himiko](https://osu.ppy.sh/users/1806962)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/loving.jpg)](https://osu.ppy.sh/beatmapsets/1080025)

'How on earth did eroge get into spotlights?' may be a question that pops up when first encountering this set. Despite the general stigma against the genre, [Luminate](https://osu.ppy.sh/users/4778689)'s "Loving Trip" does a great job of following the song. 

Slower parts of the song keep things interesting with varied rhythm and patterns, while higher density sections have liberal use of antiflow that contrasts well. This is one of the few sets that I've genuinely had fun playing this year, and I hope you give it a shot as well.

> *written by [Sinnoh](https://osu.ppy.sh/users/4236057)*

Reviewer 4: 100% missed pun opportunity for 'visually novel'

Reviewer 5: not interested to apply this

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/lights.jpg)](https://osu.ppy.sh/beatmapsets/1044462)

[Kukkai](https://osu.ppy.sh/users/7811952)'s "Lights of Muse" has a certain feeling behind it that is fairly uncommon these days, reminiscent of something that would have easily had a play count in the hundred thousand if it was ranked a few years ago. While the use of patterns and rhythm is fairly simple, the whole set is at a consistent level of fun that is hard to come by.

*written by [Sinnoh](https://osu.ppy.sh/users/4236057)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/sweet.jpg)](https://osu.ppy.sh/beatmapsets/738169)

> [Sorcerer](https://osu.ppy.sh/users/1913190) and [ZiRoX](https://osu.ppy.sh/users/200768) has something sweet to offer in this season's Spotlights!

Reviewer 4: ```suggestion
[Sorcerer](https://osu.ppy.sh/users/1913190) and [ZiRoX](https://osu.ppy.sh/users/200768) have something sweet to offer in this season's Spotlights!
```

> Starting from the bottom we have Cup, Salad and Platter all made by Sorcerer, and they lead the player through the learning stages of the streams and how to control the jumps. If you can already get full combo on the Salad then the next step is to learn how to handle the Hyperjumps and time your normal jumps at the end of the streams in Platter. Very well designed difficulty curve.

Reviewer 4: you can probably condense this into one sentence about difficulty curve

Reviewer: ```suggestion
Starting from the bottom we have Cup, Salad and Platter all made by Sorcerer, and they lead the player through the learning stages of the streams and jumps. Platter introduces Hyperjumps and even more advanced stream patterns.
```

Next up we have two Rain to choose from, one from the mapper of the previous three and one guest difficulty from ZiRoX. Dispite their similar star ratings there is a clear difference in them. ZiRoX decided to use less Hyperjump during the stream parts but instead use stair patterns, which are not necessarily easier to handle though. Meanwhile Sorcerer decided to step up from Platter and make his Hyperjump streams even more deadly for the player with more experience.

> And for last we have a Sweet Overdose, which is not really that sweet if you take a peek inside. It raises to jump stream pattern into a whole new level where the players have to face the true evil in case they dare to challenge it. The sweetness flew away and only Rain remained, raining fruits and Hypers everywhere. Prepare your umbrella and get ready to catch 'em all!

Reviewer 4: in general less description about what the set is and more description about the themes/techniques it explores is better

Reviewer: That is not easy to do when the set is all about one technique and stays consistent though. I am not sure if this one is better either.
```suggestion
Next up we have two Rain to choose from, one from the mapper of the previous three and one guest difficulty from ZiRoX. Despite their similar star ratings, there is a clear difference in them. ZiRoX uses less Hyperjump during the stream parts but went for stair patterns and some quick 1/4 slider jump pattern later on. Meanwhile, Sorcerer decided to step up from Platter and make his Hyperjump streams even more deadly for the player with more experience.

And for last we have a Sweet Overdose which has short slider Hyperdashes to follow the melody. It also uses the same reverse slider pattern from rain to temporarily stop the player's movement. The flow is usually straightforward, but sometimes you might find some antiflow pattern. The sweetness flew away and only Raining fruits remained. Prepare your umbrella and get ready to catch 'em all!
```

*written by [Kurokami](https://osu.ppy.sh/users/260933)*

## <a id="mania"></a> osu!mania

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/mania/transhuman.jpg)](https://osu.ppy.sh/beatmapsets/1069426)

> Incredibly exciting yet surprisingly delicate in its structure, [Kamuy](https://osu.ppy.sh/users/7439226)'s Transhumanist stands out among currently ranked maps by being the first semi-dump ranked in osu!mania. This map has been making waves in both mapping as well as higher-level playing communities by combining consistent usage of pattern/snap arrangements to represent specific kinds of wubs in the song in order to fully represent intensity with player-friendly yet challenging patterns. These make it an absolute joy to play through and fascinating to pick apart in the editor as well.

Reviewer 4: what does semi-dump mean exactly?

Reviewer 6: It's a rather ambiguous term? Dumps are usually described as placing notes where there isn't a distinct - yet continuous - sound, but it still flows with the music; it typically uses certain snaps to represent the sound(s). Because of this, it is one of the reasons why it can be hard to describe what a dump exactly is. Semi-dump is pretty much where you only do this for specific sounds (for example in this map, the dump is exclusive to the wubs in the song).

> Even though it's the first of it's kind in the ranked section, Transhumanist sets an extremely high bar and proves that these kinds of maps are not only rankable, but exceptional in terms of mapping quality and providing an enjoyable player experience. This map is arguably one of Kamuy's best maps so far and we're proud to showcase it in spotlights!

Reviewer 6: ```suggestion
Even though it's the first of its kind in the ranked section, Transhumanist sets an extremely high bar and proves that these kinds of maps are not only rankable, but exceptional in terms of mapping quality and providing an enjoyable player experience. This map is arguably one of Kamuy's best maps so far and we're proud to showcase it in spotlights!
```

*written by [-MysticEyes](https://osu.ppy.sh/users/6253266)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/mania/experiment.jpg)](https://osu.ppy.sh/beatmapsets/914341)

[Curiossity](https://osu.ppy.sh/users/6490035) strikes again with another insanely innovative and creative masterpiece.

Every bit of this map is thought out with very particularly detailed and visually appealing patterning and speed changes, which gives the map a great amount of replay value, especially for a marathon. The map's excitement is maintained throughout the map, making for a roller-coaster of an experience. 

> It never gets boring, not even during the slower sections, in which the player must pay very close attention to the long note releases at a slow scroll speed. This map thus deserves its top spot in the spotlights, and we hope to see Curiossity continue to produce spectacular maps like this one in the future.

Reviewer 6: ```suggestion
It never gets boring, not even during the slower sections, in which the player must pay very close attention to the long note releases at a slow scroll speed. This map thus deserves its spot in the spotlights, and we hope to see Curiossity continue to produce spectacular maps like this one in the future.
```
it's not the top spot, so the adjective here is unnecessary

If you had to describe this map with one word, it'd be something along the lines of... Intricate. Detailed. Accurate. Clean. Recently we have seen a rise of mappers in osu!mania with this approach in mind, [Monheim](https://osu.ppy.sh/users/6814203) and [Vortex-](https://osu.ppy.sh/users/4999669) being well-known for their precision and accuracy in their maps. 

This map, in particular, accomplishes things that many mappers most likely would not be able to accomplish if they were to map this - it is musically accurate in that it expresses many instruments and melody lines at once, is delicate and clean to play, and provides a challenge for all sorts of players. These remain consistent through each difficulty of the mapset. This map thus deserves its spot in this season's spotlights, and be on the lookout - we most definitely will see more by these two extraordinarily talented mappers in the future.

*written by [-mint-](https://osu.ppy.sh/users/8976576)*

[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/mania/oceanus.jpg)](https://osu.ppy.sh/beatmapsets/539179)

> It has been a while before [Level 51](https://osu.ppy.sh/users/3617847) made his appearance in mapping ground, and this time he brings the community with a great 6k mapset, Oceanus. 

Reviewer 4: ```suggestion
It has been a while since [Level 51](https://osu.ppy.sh/users/3617847) made his appearance in the mapping scene, and this time he brings the community a great 6k mapset, Oceanus. 
```

> GALE_FORCE, as the top difficulty of the mapset, proposes a real challenge for advanced players with its dense but well-balanced streams that follow the shifts in the music. On the other hand, the lower two difficulties definitely provide great opportunities for general players to start with 6K and practice over 6K streams as the mapper managed to effectively balance the density within and between these difficulties. 

Reviewer 4: ```suggestion
GALE_FORCE, as the top difficulty of the mapset, proposes a real challenge for advanced players with its dense but well-balanced streams that follow the shifts in the music. On the other hand, the lower two difficulties definitely provide great opportunities for general players to start with 6K and practice over 6K streams as the mapper has managed to effectively balance the density within and between these difficulties. 
```
